scripts/boilerplate.go.txt

```diff
@@ -0,0 +1,13 @@
+// Copyright The Kubernetes Authors.
```

Comment:
Just from interest, what is the purpose of boilerplate.go.txt file?

Reply:
It's provided to `openapi-gen` via the `-h` option to add a license header to `pkg/api/generated/openapi/zz_generated.openapi.go`